src/steamship/base/client.py

```diff
@@ -188,12 +188,28 @@ def dict(self, **kwargs) -> dict:
         # implementations), Pydantic will try to include them by default. So we have to suppress that otherwise
         # downstream serialization into JSON will fail.
         if "exclude" not in kwargs:
-            kwargs["exclude"] = {"use", "use_plugin", "_instance_use", "_instance_use_plugin"}
+            kwargs["exclude"] = {
+                "use": True,
+                "use_plugin": True,
+                "_instance_use": True,
+                "_instance_use_plugin": True,
+                "config": {"api_key"},
```

Comment:
This seems fine to me -- I can't think of anywhere we depend on this, and arguably nobody should ever want this in the dict serialization